README.md

```diff
@@ -1,70 +1,192 @@
+To get a local copy up and running, follow these steps.
 
-### Making a Progressive Web App
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+In order to run this project you need:
 
-### Advanced Configuration
+- Nodejs
+- Node Package Manager
+- VS Code text editor
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Setup
 
-### Deployment
+Clone this repository to your desired folder:
+```sh
+cd my-folder
+git clone git@github.com/touredri/fruit.git
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Install
+Install this project with:
+```sh
+cd my-project
+npm install
+```
 
-### `npm run build` fails to minify
+### Usage
+To run the project, execute the following command:
+```bash
+  npm start
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Run tests
+To run tests, run the following command:
+
+#### Jest Test
+```bash
+  npm test
+  ```
+
+#### Linters
+```bash
+npx eslint "**/*.{js,jsx}"
+```
+or 
+```bash
+npx eslint "**/*.{js,jsx}" --fix
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+#### 👤 **Drissas Touré**
+
+- Github: [Drissas](https://github.com/touredri)
+- Twitter: [Drissas](https://twitter.com/touredri)
+- LinkedIn: [Drissas](https://www.linkedin.com/in/touredri/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add record view
+- Add possibility to share
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contribute <a name="contribute"></a>
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues](https://github.com/touredri/fruit/issues) page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project then please give a ⭐️ and also share your feedback.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## Acknowledgments
+Big ups to Microverse team .
+Thanks to https://www.fruityvice.com/ for their API
+Thanks to Unsplash: https://help.unsplash.com/en/articles/2511271-guideline-hotlinking-images also for they image API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Kindly provide appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
Ok @frank1738